code/game/objects/structures/stairs.dm

```diff
@@ -52,13 +53,25 @@
 
 	AddElement(/datum/element/connect_loc, loc_connections)
 
+	if(mapload)
+		resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
```

Comment:
> INDESTRUCTIBLE

No.

Reply:
armor is stupid shit and a value of 100 still doesnt protect it from said damage so what do i even do here